.githooks/commit-msg/check-for-changelog-lint

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Because commits are hard to remove (unless you are a pro at interactive
+# rebasing), this commit hook script should make it easier for new
+# contributors.
+
+commit_msg=$1
+cat "$commit_msg" | npm run changelog-lint-from-stdin
```

Comment:
@kumar303 how about adding the `--silent` npm run option here? e.g.

```
cat "$commit_msg" | npm run changelog-lint-from-stdin --silent
```

so that we can get a less cluttered and less confusing error message (because it doesn't include the verbose default error message from the "npm run" failure):

```
$ git commit -m "invalid commit message"

⧗   input: invalid commit message
✖   message may not be empty [subject-empty]
✖   type may not be empty [type-empty]
✖   type must be one of ["chore", "docs", "feat", "fix", "perf", "refactor", "style", "test"] [type-enum]
✖   found 3 problems, 0 warnings

Your unsaved commit message:

invalid commit message

--------------------------------------------------------------------
Your commit was not saved because the message was formatted incorrectly.
See the error log above.
--------------------------------------------------------------------

For formatting instructions, see:
https://github.com/mozilla/web-ext/blob/master/CONTRIBUTING.md#writing-commit-messages

To bypass this check and commit anyway, use:
git commit --no-verify ...

$
```